stock_picking_rate/tests/test_stock_picking_rate_purchase.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from .common import TestHelper
+
+
+class TestStockPickingRate(TestHelper):
+
+    def setUp(self):
+        super(TestStockPickingRate, self).setUp()
+        self.rate_ids = [self.new_record(), self.new_record()]
+        self.Wizard = self.env['stock.picking.rate.purchase']
+        self.wizard_vals = {
+            'rate_ids': [(6, 0, [r.id for r in self.rate_ids])],
+        }
+
+    def new_wizard(self, vals=None):
+        vals = self.wizard_vals if vals is None else {}
```

Comment:
This method is not getting used anywhere. Also, the first line seems wrong. As it stands, if the call includes an argument for `vals`, that argument won't be used and `vals` will simply be set to an empty dictionary.